packages/legend-extension-dsl-persistence/src/graph-manager/protocol/pure/v1/model/packageableElements/persistence/V1_DSL_Persistence_ActionIndicatorFields.ts

```diff
@@ -32,9 +33,11 @@ export abstract class V1_DeleteIndicator extends V1_ActionIndicatorFields {
 }
 
 export class V1_DeleteIndicatorForGraphFetch extends V1_DeleteIndicator {
+  deleteFieldPath!: object;
```

Comment:
can you add the comment like we have in other places? something like

```ts
  /**
   * Studio does not process value specification, they are left in raw JSON form
   *
   * @discrepancy model
   */
```

Reply:
Updated.